master.yml

```diff
@@ -60,15 +60,14 @@ Parameters:
 
   #  ############## Service #############
 
-  NetworkName:
+  CasperNetwork:
     Type: String
-    Default: delta-11
-    Description: Name of the network
+    Default: casper
```

Comment:
I think this should default to casper-test rather than casper, as we want their first join to a TestNet rather than MainNet I think.

Reply:
Fixed in last commit